src/daria/image/image.py

```diff
@@ -112,25 +147,21 @@ def __init__(
                 "Invalid image data. Provide either a path to an image or an image array."
             )
 
-        # Read metadata from file or create from input arguments
-        if read_metadata_from_file:
-            self.create_metadata_from_file(metadata_path)
-        else:
-            self.origo = origo if isinstance(origo, np.ndarray) else np.array(origo)
-            self.width = width
-            self.height = height
-            self.dim = dim
-        self.shape = self.img.shape
+        if curvature_correction is not None:
+            self.img = curvature_correction(self.img)
+            self.width = curvature_correction.config["crop"]["width"]
+            self.height = curvature_correction.config["crop"]["height"]
+            self.origo = [0, 0]
+            self.update_metadata()
+
+        if color_correction is not None:
+            self.toRGB()
+            self.img = color_correction(self.img)
 
         # Determine numbers of cells in each dimension and cell size
-        self.num_pixels_height, self.num_pixels_width = self.shape[:2]
+        self.num_pixels_height, self.num_pixels_width = self.img.shape[:2]
         self.dx = self.width / self.num_pixels_width
         self.dy = self.height / self.num_pixels_height
-        # ... in 3d
-        if self.dim == 3:
-            self.depth = depth
-            self.num_pixels_depth = self.shape[2]
-            self.dz = self.depth / self.num_pixels_depth
```

Comment:
I agree that we should for now consider only 2d images!

Reply:
The depth can be actually very relevant in terms of the FluidFlower. But this is very specific and should not be included here. In particular, because the depth is not constant for the FluidFlower rigs.